docs/source/app-dev/services.rst

```diff
@@ -193,7 +193,25 @@ Party management service
 
 Use the **party management service** to allocate parties on the ledger and retrieve information about allocated parties.
 
-Allocating parties is necessary to interact with the ledger. For more information, refer to the pages on :doc:`Identity Management</concepts/identity-and-package-management>` and :ref:`the API reference documentation <com.daml.ledger.api.v1.admin.PartyManagementService>`.
+Parties serve to govern on-ledger access control as per :ref:`Daml's privacy model <da-model-privacy>`
+and :ref:`authorization rules <da-ledgers-authorization-rules>`.
+Applications and their operators are expected allocate and use parties to manage on-ledger access control as per their business requirements.
+
+For more information, refer to the pages on :doc:`Identity Management</concepts/identity-and-package-management>` and :ref:`the API reference documentation <com.daml.ledger.api.v1.admin.PartyManagementService>`.
+
+.. _user-service:
+
+User management service
+=======================
+
+Use the **user management service** to manage the set of users on a participant node and their :ref:`access rights <authorization-claims>` to that node's ledger API services.
```

Comment:
```suggestion
Use the **user management service** to manage the set of users on a participant node and their :ref:`access rights <authorization-claims>` to that node's Ledger API services.
```
:P

Reply:
Going to stop flagging it up here. Just need to make a decision whether to capitalize or not and go with one or the other.

Reply:
I'll search an replace to capitalized to keep the current style in the glossary.